docs/source/developer_guides/metrics.md

```diff
@@ -40,10 +40,6 @@ The following metrics are pre-registered:
 | `entropy_per_cell`       | integration      | Assesses the mixing of batch labels at the single-cell level. Higher entropy indicates better integration.       |
 | `batch_silhouette`       | integration      | Combines silhouette scoring with batch information to evaluate clustering quality while accounting for batch effects. |
 | `spearman_correlation`   | perturbation     | Rank correlation between predicted and actual values     |
-| `accuracy_score`         | perturbation     | Accuracy of binary classification predictions against ground truth labels     |
-| `f1_score`               | perturbation     | F1-score balancing precision and recall for binary classification tasks     |
```

Comment:
f1_score still exists, but is now used by cross-species-label-prediction task.

Reply:
we actually have other fixes to make here, unrelated to perturb seq task, so no need to change on this PR